third-party/realdds/py/pyrealdds.cpp

```diff
@@ -564,6 +564,7 @@ PYBIND11_MODULE(NAME, m) {
     using realdds::dds_option_range;
     py::class_< dds_option_range >( m, "dds_option_range" )
         .def( py::init<>() )
+        .def( py::init<>( []( float n, float x, float s, float def ) { return dds_option_range{ n, x, s, def }; } ) )
```

Comment:
What are `n` and `x` - min and max? If so name them accordingly

Reply:
OK...